sklearn/experimental/enable_hist_gradient_boosting.py

```diff
@@ -1,36 +1,19 @@
-"""Enables histogram-based gradient boosting estimators.
+"""This is now a no-op and can be safely removed from your code.
```

Comment:
Maybe add a comment "TODO remove in 1.2" so that we won't forget

Reply:
It was my understanding that we would basically keep these files forever to avoid any breakage?
IIRC this was the whole point of doing it with "enable" imports, instead of putting the whole code into the `experimental` module (like `from sklearn.experimantal import HistGradientBoostingClassifier`)

Reply:
I don't get why but I missed the discussions so there might be a reason I don't see. Let's keep it, there's no hurry anyway.

Reply:
@jeremiedbb I have the same question as you. Without any prior knowledge, my guess would be to remove this file at some point in the future, preferably 1.2.

Reply:
IIRC this was mostly discussed during the 2019 paris sprint. Also in https://github.com/scikit-learn/scikit-learn/issues/13499 and probaby in https://github.com/scikit-learn/scikit-learn/pull/12807

Not breaking user code when a feature becomes stable was a major concern, @ogrisel may be able to confirm.

IMHO it doesn't hurt to leave that file there indefinitely, we're not going to put the estimators back into experimental

Reply:
Thanks for the explanations. This would be valuable info in the dev guide.